tests/factories/test-factories.js

```
@@ -0,0 +1,531 @@
+			global.testFactories = TestFactories;
+			global.mockServiceRegistry = MockServiceRegistry;
+		}
+	}
+
+	/**
+	 * Teardown test environment
+	 */
+	static teardown() {
+		// Clean up globals if they exist
+		if (typeof global !== 'undefined') {
+			delete global.testFactories;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)